hub/db/db.py

```diff
@@ -494,13 +494,25 @@ def get_claim_txo(self, claim_hash: bytes) -> Optional[ClaimToTXOValue]:
         return self.prefix_db.claim_to_txo.get(claim_hash)
 
     def _get_active_amount(self, claim_hash: bytes, txo_type: int, height: int) -> int:
+        if height == self.db_height + 1:
+            v = self.prefix_db.effective_amount.get(claim_hash)
+            if not v:
+                return 0
+            if txo_type is ACTIVATED_SUPPORT_TXO_TYPE:
+                return v.support_sum
+            return v.effective_amount - v.support_sum
```

Comment:
As we chatted about, I would like to see the two fields named something that indicates `support_sum` is included in the other one.

`activated_sum` `activated_support_sum`
`total_amount` `support_amount`

...or something along those lines. `effective_amount` doesn't immediately call to mind (claim amount + support amounts) for me as a newbie.